nengo_spa/algebras/hrr_algebra.py

```diff
@@ -82,6 +126,48 @@ def bind(self, a, b):
             raise ValueError("Inputs must have same length.")
         return np.fft.irfft(np.fft.rfft(a) * np.fft.rfft(b), n=n)
 
+    def binding_power(self, v, exponent):
+        r"""Returns the binding power of *v* using the *exponent*.
+
+        The binding power is defined as binding (*exponent*-1) times bindings
+        of *v* to itself. Fractional binding powers are supported.
+
+        Note the following special exponents:
+
+        * an exponent of -1 will return the inverse,
+        * an exponent of 0 will return the identity vector,
+        * and an *exponent* of w1cne will return *v* itself.
+
+        The following relations hold for integer exponents, and for unitary
+        vectors:
+
+        * :math:`\mathcal{B}(v^a, v^b) = v^{a+b}`,
+        * :math:`(v^a)^b = v^{ab}`.
```

Comment:
Could mention the first one holds for real exponents too. :)

Reply:
* [ ] Not obvious to me right now, but will double check that this is true and add the mention (especially because it might be non-obvious).

Reply:
Given non-unitary vectors, this actually only holds for **non-negative** real exponents. A negative exponent implies an approximate inverse (at least within Nengo SPA) and that invalidates the relationship.